contracts/tests/sbtc-token.test.ts

```diff
@@ -87,5 +87,52 @@ describe("sBTC token contract", () => {
       expect(receipt2.value).toEqual(999n);
     });
 
+    test("Fail a non-protocol principal calling protocol-mint", () => {
+      const receipt = txErr(
+        token.protocolMint({
+          amount: 1000n,
+          recipient: bob,
+        }),
+        bob
+      );
+      expect(receipt.value).toEqual(token.constants.ERR_NOT_AUTH.value);
+    });
+
+    test("Fail transferring sbtc when not owner", () => {
+      const receipt = txOk(
+        deposit.completeDepositWrapper({
+          txid: new Uint8Array(32).fill(0),
+          voutIndex: 0,
+          amount: 1000n,
+          recipient: alice,
+        }),
+        alice
+      );
+      const printEvents = filterEvents(receipt.events, CoreNodeEventType.ContractEvent);
+      const [print] = printEvents;
+      const printData = cvToValue<{
+        topic: string;
+        txid: string;
+        voutIndex: bigint;
+        amount: bigint;
+      }>(print.data.value);
+      expect(printData).toStrictEqual({
+        topic: "completed-deposit",
+        txid: new Uint8Array(32).fill(0),
+        voutIndex: 0n,
+        amount: 1000n,
+      });
```

Comment:
Nit: I think we can just remove this assertion since we already have it in other places, and it's just cleaner to keep each individual test scoped

Reply:
Removed!